### ISSUE TITLE
爱可可微博热门分享(3.31)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#114]
No 1. <a href="https://weibo.com/1402400261/O7wM56Ev8">《LLM 应用开发实践笔记》</a>
No 2. <a href="https://weibo.com/1402400261/O7w5scEbo">《混合专家模型 (MoE) 详解》</a>
No 3. <a href="https://weibo.com/1402400261/O7wxgeqRs">对 预训练 MoE、upcycled MoE 和 FrankenMoE 三类 MoE 模型的简介。 - 预训练 MoE 预训练 ......</a>
No 4. <a href="https://weibo.com/1402400261/O7ye0lEsa">LLocalSearch：使用LLM Agent完全本地运行的元搜索引擎，用户可以问一个问题，系统将使用LLM链来找到答案，用户可以看到agent的进度和最终的答案，不需要OpenAI或Google API密钥</a>
No 5. <a href="https://weibo.com/1402400261/O7vOKCeLv">DSPy：机器学习工作流重塑提示工程</a>
No 6. <a href="https://weibo.com/1402400261/O7yyxwwUe">AI生成内容泛滥，文化生态岌岌可危</a>
No 7. <a href="https://weibo.com/1402400261/O7xaJE7HH">Plandex：基于OpenAI API的命令行AI编程引擎，用于处理复杂的任务，可以帮助用户分解大型任务为更小的子任务，并逐个实现这些子任务，帮助快速完成工作流，与不熟悉的技术互动，摆脱困境，减少在无聊的事情上花费的时间</a>
No 8. <a href="https://weibo.com/1402400261/O7yfei9ug">高效SAM分割模型大列表</a>
No 9. <a href="https://weibo.com/1402400261/O7ys0ziZp">Edge Infer：- 旨在资源受限的设备上运行小型 AI 模型(包括向量化和Onnx模型)，如 Android、iOS 或 MCU，实现高效的边缘智能，用于实时决策</a>
No 10. <a href="https://weibo.com/1402400261/O7yttipiX">EPUB to Audiobook Converter：EPUB到有声书的转换器</a>
No 11. <a href="https://weibo.com/1402400261/O7uxHpnbg">[LG]《Visualizing High-Dimensional Temporal Data Using Direction-Aware t-SNE》</a>
No 12. <a href="https://weibo.com/1402400261/O7n3vpf2O">开源大语言模型的选择之道</a>
No 13. <a href="https://weibo.com/1402400261/O7uEusOfo">[CL]《What Are Tools Anyway? A Survey from the Language Model Perspective》</a>
No 14. <a href="https://weibo.com/1402400261/O7x2Kl9x2">OpenDevin: 一个开源项目，旨在复制Devin，Devin是一个自主的AI软件工程师，能执行复杂的工程任务，并积极与用户协作进行软件开发项目的合作</a>
No 15. <a href="https://weibo.com/1402400261/O7lvlzpnR">#免费##抽奖# 携手@博文视点Broadview 送出3本《码农翻身2》</a>
No 16. <a href="https://weibo.com/1402400261/O7yt26gby">Softmax for Arbitrary Label Trees：一个用于医学影像分割的框架</a>
No 17. <a href="https://weibo.com/1402400261/O7x9Clzzn">深度伪造(Deepfake)相关文献资源列表</a>
No 18. <a href="https://weibo.com/1402400261/O7rJnwv4U">晚安～ [月亮] ​</a>
No 19. <a href="https://weibo.com/1402400261/O7mnVgkA5">不确定的确定性：用生成式模型颠覆天气预报</a>
No 20. <a href="https://weibo.com/1402400261/O7hrKuBtP">RAG相关资源大列表</a>
No 21. <a href="https://weibo.com/1402400261/O7yrtjJXR">hoarder-app：用于数据(链接、图片、笔记)收集的应用，支持AI自动打标和全文检索功能</a>
No 22. <a href="https://weibo.com/1402400261/O73tVBTUS">如何成为“效率大师”？</a>
No 23. <a href="https://weibo.com/1402400261/O7ynd9CZx">rev.ng：基于LLVM和QEMU的二进制分析框架和反编译器</a>
No 24. <a href="https://weibo.com/1402400261/O7lcEBZ31">[CL]《sDPO: Don't Use Your Data All at Once》</a>
No 25. <a href="https://weibo.com/1402400261/O7xdOmrEj">RestAI：基于LlamaIndex、Ollama和HF pipelines的AIaaS(人工智能即服务)开源平台</a>
No 26. <a href="https://weibo.com/1402400261/O7ycr287E">Vector IO：使用一种通用的向量数据集格式，以轻松地从所有矢向量数据库中导出和导入数据</a>
No 27. <a href="https://weibo.com/1402400261/O7uvzAOpA">早！[太阳]</a>
No 28. <a href="https://weibo.com/1402400261/O7uxTeDlB">本文通过在t-SNE中加入方向一致性和边长惩罚，产生能够突出时间模式的低维嵌入。//[LG]《Visualizing High-Dimensional Temporal Data Using Direction-Aware t-SNE》</a>
No 29. <a href="https://weibo.com/1402400261/O7uVh0dhT">今日推介(第1361期)：使用方向感知t-SNE可视化高维时间数据、超越回路重叠寻找模型机制、语言模型视角下的工具使用研究综述、用引导扩散从零开始生成强大的投毒和后门、用于逼真目标移除和插入的自举反事实预训练 ......</a>
No 30. <a href="https://weibo.com/1402400261/O7ym7486g">EasyRL4Rec - 专注于强化学习(RL)为基础的推荐系统(RS)的全面且易于使用的库</a>
No 31. <a href="https://weibo.com/1402400261/O7ypPED6c">Popple：分布式、高可用、通用用途的键/值数据库</a>
No 32. <a href="https://weibo.com/1402400261/O7uR2hqNe">[CV]《ObjectDrop: Bootstrapping Counterfactuals for Photorealistic Object Removal and Insertion》</a>
No 33. <a href="https://weibo.com/1402400261/O7lGhyHkc">Voice Engine：15秒音频样本，开启逼真语音合成新时代</a>
No 34. <a href="https://weibo.com/1402400261/O7uEwqhRZ">通过定义工具的概念、总结应用场景和方法、分析效率等方面，全面系统地回顾和分析了工具辅助语言模型的研究进展，为该领域的发展提供了重要指导。//[CL]《What Are Tools Anyway? A Survey from the Language Model Perspective》</a>
No 35. <a href="https://weibo.com/1402400261/O7gdbAQPy">'《一人企业方法论》</a>
No 36. <a href="https://weibo.com/1402400261/O7x5h1h9B">OpenUI：让使用者描述网页界面并实时渲染的开源项目</a>
No 37. <a href="https://weibo.com/1402400261/O7bJ6kisn">[LG] Tutorial on Diffusion Models for Imaging and Vision 本教程全面系统地介绍了从变分自编码器到扩散模型的发展脉络，数学推导细致，内容丰富，是理解扩散模型的好教材。 ......</a>
No 38. <a href="https://weibo.com/1402400261/O7hvwvhmu">Bolna：快速构建LLM语音聊天应用的关键框架，帮助开发人员轻松打造高效的聊天应用</a>
No 39. <a href="https://weibo.com/1402400261/O7uKD6OzN">利用引导扩散生成高质量和高效的中投毒基础样本，再与下游攻击算法相结合，实现了更隐蔽和强效的对抗机器学习。//[LG]《Generating Potent Poisons and Backdoors from Scratch with Guided Diffusion》</a>
No 40. <a href="https://weibo.com/1402400261/O7lI1oWfy">三层评估法则：快速迭代，系统优化AI产品</a>
No 41. <a href="https://weibo.com/1402400261/O6MDExdGc">A Survey of LLM Surveys：精心分类的LLM综述大列表</a>
No 42. <a href="https://weibo.com/1402400261/O7bLx6SZP">[CL] BioMedLM: A 2.7B Parameter Language Model Trained On Biomedical ......</a>
No 43. <a href="https://weibo.com/1402400261/O7gMqyvjl">DeepSeek：基于LLM的检索引擎</a>
No 44. <a href="https://weibo.com/1402400261/O7l3GfFP2">[LG]《Model Stock: All we need is just a few fine-tuned models》</a>
No 45. <a href="https://weibo.com/1402400261/O7llHvZry">[LG] Tiny Machine Learning: Progress and Futures TinyML通过算法和系统的协同设计，实现深度学习模型在内存和计算有限的微控制器上的高效推理和训练，将强大的AI能力扩展到无数的边缘物联网设备中。 ......</a>
No 46. <a href="https://weibo.com/1402400261/O6YA4eBGV">Sora运行揭秘</a>
No 47. <a href="https://weibo.com/1402400261/O6W8b6Y2a">LLM-Culture：用于研究大型语言模型文化演化的开源框架</a>
No 48. <a href="https://weibo.com/1402400261/O7uR4D6Cm">通过构建反事实图像数据集微调扩散模型，实现了逼真的物体删除和插入，显著提高了渲染物体对场景效应的能力。//[CV]《ObjectDrop: Bootstrapping Counterfactuals for Photorealistic Object Removal and Insertion》</a>
No 49. <a href="https://weibo.com/1402400261/O7m5fdezH">Maxime Labonne：Starling-LM-7B-beta模型在各种基准测试中表现出色，这给我们带来了一些启示</a>
No 50. <a href="https://weibo.com/1402400261/O7lvdpHBP">今日推介(第1360期)：只用少数微调模型达到模型融合性能、教语言模型提问澄清问题、直接偏好优化中长度与质量的解缠、将偏好数据集分割逐步使用的DPO、多专家递延回归 ......</a>
